internal/accounts/contract-update.go

```diff
@@ -50,7 +51,7 @@ var UpdateCommand = &command.Command{
 func updateContract(
```

Comment:
Update shouldn't accept init args since you can't upgrade contract with init.

Reply:
addressed